docroot/modules/custom/moj_resources/src/SeriesContentApiClass.php

```diff
@@ -225,9 +185,41 @@ private function getSeriesContentNodeIds($seriesId, $number, $offset, $prison)
     $results = $this->entity_query->get('node')
       ->condition('status', 1)
       ->accessCheck(false);
-
     $results->condition('field_moj_series', $seriesId);
 
+    $seriesTerm = Term::load($seriesId);
+    $sortByFieldValue = $seriesTerm->get('field_sort_by')->getValue();
+    $sortByFieldValue = empty($sortByFieldValue) ? NULL : $sortByFieldValue[0]['value'];
+
+    $sortFields = [];
+    $sortDirection = '';
```

Comment:
Are these two lines needed? It looks like the switch statement will always set a value, so these lines may be redundant.

Reply:
Ah yeah that was more from a general habit of setting up variables with default values before setting them.  But you're right as there is the `default` in the switch statement, these aren't needed.  I've removed them.